crates/node-bindings/Cargo.toml

```diff
@@ -58,7 +58,7 @@ rayon = "1.7.0"
 getrandom = { version = "0.2", features = ["custom"], default-features = false }
 napi = { version = "2.16.4", features = ["serde-json"] }
 
-[target.'cfg(target_os = "macos")'.dependencies]
+[target.'cfg(all(target_os = "macos", not(feature = "canary")))'.dependencies]
```

Comment:
Maybe just leave a comment here to specify why it's being disabled for canary?

Reply:
Added a comment